src/components/ListPullRequests.vue

```diff
@@ -1,53 +1,102 @@
 <template>
   <div class="list-pull-request">
-    <div class="pr-title">Pull Requests List</div>
+    <div class="pr-title">Hacktober Fest 2019</div>
     <div class="pr-list">
       <div
         v-for="(user, userIndex) in users"
-        v-bind:key="user.username"
+        v-bind:key="userIndex"
```

Comment:
There is a reason for using the index here instead of username?
Because this can cause bug since Vue will use it to render or not again the item in the list.

For example, if I'm rendering a list of texts with the array `['apple', 'banana', 'pineapple']` 
```
- apple (key: 0)
- banana (key: 1)
- pineapple (key: 2)
```
and I change it to `['apple', 'pineapple']` the second item will render `banana` since the key is the same.
```
- apple (key: 0)
- banana (key: 1)
```
;)